langchain/src/tools/outlook/authFlowToken.ts

```diff
@@ -0,0 +1,103 @@
+import { AuthFlowBase } from "./authFlowBase.js";
+import { getEnvironmentVariable } from "../../util/env.js";
+
+interface AccessTokenResponse {
+  access_token: string;
+  refresh_token: string;
+}
+
+// if you have the token, and no need to refresh it, warning: token expires in 1 hour
```

Comment:
Convert to JSDoc and give more of an overview as to what this class can be used for